README.md

```diff
@@ -40,5 +45,34 @@
   > bundle:start <net.staticsnow:nexus-repository-apt ID>
   ```
 
+#### (more) Permanent Install
+
+For more permanent installs of the nexus-repository-apt plugin, follow these instructions:
+
+* Copy the bundle (nexus-repository-apt-1.0.2.jar) into <nexus_dir>/deploy
+
+This will cause the plugin to be loaded with each restart of Nexus Repository. As well, this folder is monitored
+by Nexus Repository and the plugin should load within 60 seconds of being copied there if Nexus Repository
+is running. You will still need to start the bundle using the karaf commands mentioned in the temporary install.
+
+#### (most) Permanent Install
+
+If you are trying to use the APT plugin permanently, it likely makes more sense to do the following:
+
+* Copy the bundle into `<nexus_dir>/system/net/staticsnow/nexus-repository-apt/1.0.2/nexus-repository-apt-1.0.2.jar`
+* Add the following lines to `<nexus_dir>/system/com/sonatype/nexus/assemblies/nexus-oss-feature/3.x.y/nexus-oss-feature-3.x.y-features.xml`
+  ```
+    <feature prerequisite="false" dependency="false">nexus-repository-apt</feature>
```

Comment:
Some context of where these lines should go would be helpful. e.g. this line should go just inside the `<feature name="nexus-oss-feature" ...` with the other format plugins, and the next set of lines can go at the end just before the closing `</features`